insights/parsers/ceilometer_log.py

```diff
@@ -32,7 +32,21 @@ class CeilometerCentralLog(LogFileOutput):
     Note:
         Please refer to its super-class ``LogFileOutput``
     """
-    pass
```

Comment:
Yep - that was already in the documentation for that parser, I didn't include it :-)